dali/base/dautils.cpp

```diff
@@ -3321,22 +3324,30 @@ class CLocalOrDistributedFile: implements ILocalOrDistributedFile, public CInter
                 getDefaultStoragePlane(cluster);
             Owned<IStoragePlane> plane = getDataStoragePlane(cluster, true);
             dir.append(plane->queryPrefix());
+            unsigned numStripedDevices = plane->numDevices();
+            if (numStripedDevices>1)
+                stripeNum = calcStripeNumber(0, lfn.get(), numStripedDevices);
 #endif
+            StringBuffer descPath;
+            makePhysicalPartName(lfn.get(), 0, 0, descPath, 0, DFD_OSdefault, dir, false, false);
```

Comment:
Last parameter should be a stripeNumber rather than a boolean.  Is it worth having a special function for this since similar code occurs elsewhere (I assume it gets the base directory).